internal/net/net.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-2021 MinIO, Inc.
```

Comment:
```suggestion
// Copyright (c) 2015-2023 MinIO, Inc.
```
Faster than Harsha again :D